app/services/PageService.scala

```diff
@@ -21,7 +21,7 @@ case class DirectoryMeta(label: Option[String] = None, description: Option[Strin
 
 case class TagsModule(tags: Seq[String])
 
-case class ScenariosModule(project: Option[String] = None, branchName: Option[String] = None, feature: Option[String] = None, select: Option[TagsModule] = None)
+case class ScenariosModule(project: Option[String] = None, branchName: Option[String] = None, feature: Option[String] = None, select: Option[TagsModule] = None, includeBackground: Option[Boolean] = None)
```

Comment:
Where do you set includeBackground?

Reply:
In the scenario inclusion see https://github.com/KelkooGroup/theGardener/issues/64